pkg/reading/models.go

```diff
@@ -2,11 +2,21 @@ package reading
 
 import "github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/assets"
 
-type VersionStatus struct {
+type VersionReadStatusResponse struct {
 	Status string `json:"status"`
 }
 
-type InResponse struct {
-	Version  VersionStatus   `json:"version"`
-	Metadata assets.Metadata `json:"metadata"`
+type VersionReadIssueResponse struct {
+	Issues []string `json:"issues"`
+}
+
+// TODO to complete. This struct is use to output to a json file as the output of the resource within a concourse pipeline
+type InResponseStatus struct {
+	Version  VersionReadStatusResponse `json:"version"`
+	Metadata assets.Metadata           `json:"metadata"`
+}
+
+type InResponseIssue struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type InResponseIssue should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5050785)